Mathlib/MeasureTheory/Measure/Haar/Unique.lean

```diff
@@ -326,7 +366,7 @@ lemma haarScalarFactor_eq_mul (μ' μ ν : Measure G)
   · simp [haarScalarFactor, hG]
   -- Fix some nonzero continuous function with compact support `g`.
   obtain ⟨⟨g, g_cont⟩, g_comp, g_nonneg, g_one⟩ :
-    ∃ (g : C(G, ℝ)), HasCompactSupport g ∧ 0 ≤ g ∧ g 1 ≠ 0 := exists_continuous_nonneg_pos 1
+  ∃ (g : C(G, ℝ)), HasCompactSupport g ∧ 0 ≤ g ∧ g 1 ≠ 0 := exists_continuous_nonneg_pos 1
```

Comment:
I think I prefer the previous spacing.

Reply:
I still think I prefer the previous spacing. Could you please revert your change here?